test/TrueLayer.AcceptanceTests/PaymentsProvidersTests.cs

```diff
@@ -54,5 +59,27 @@ public async Task Can_get_payments_provider_with_mandates_capabilities()
             response.Data.Capabilities.Mandates?.VrpSweeping.ShouldNotBeNull();
             response.Data.Capabilities.Mandates?.VrpSweeping?.ReleaseChannel.ShouldNotBeNullOrWhiteSpace();
         }
+
+        [Fact]
+        public async Task Can_search_payments_providers()
```

Comment:
1. Do we have other kind of test types within the project? 
2. Are the assertions here enough?

Reply:
For now they are good enough. Unfortunately we have to test that the implementation is correct more than other internal logic, so we can't really mock some of the APIs and this type of integration tests are the only ones we can do. If the API accepts different types of parameters, we should add all of them to the test suite, unless they are pointless to test as they don't involve any internal (to this lib) logic or model mapping.